js/multimodaltextbox/shared/MultimodalTextbox.svelte

```diff
@@ -244,7 +246,7 @@
 		<Upload
 			bind:this={upload_component}
 			on:load={handle_upload}
-			filetype={accept_file_types}
```

Comment:
@freddyaboulton was this removed by accident?

Reply:
Yes, let me add back!

Reply:
Fixed